suitesparse_catkin/cmake/suitesparse_catkin-extras.cmake.in

```diff
@@ -10,7 +10,7 @@ list(APPEND @PROJECT_NAME@_LIBRARIES
   @CATKIN_DEVEL_PREFIX@/lib/libamd${CMAKE_STATIC_LIBRARY_SUFFIX} 
   lapack
   blas
-  @CATKIN_DEVEL_PREFIX@/lib/libsuitesparseconfig${CMAKE_STATIC_LIBRARY_SUFFIX} )
+  @CATKIN_DEVEL_PREFIX@/lib/libsuitesparse_catkinconfig${CMAKE_STATIC_LIBRARY_SUFFIX} )
```

Comment:
This whole file has me really skeezed out. It doesn't seem like there's anything happening in here which shouldn't be possible to achieve within the usual `catkin_package` call.

Does this even work from the installspace?